api/inc/halt_exports.h

```diff
@@ -53,4 +53,42 @@ typedef enum {
     __THALTERROR_MAX /* always keep as the last element of the enum */
 } THaltError;
 
+/* A pointer to this structure will be given to halt_error() handler
+ * of the debug box driver. */
+typedef struct {
+    /* A basic exception stack frame that is always present with a valid stack. */
+    struct {
+        uint32_t r0;
+        uint32_t r1;
+        uint32_t r2;
+        uint32_t r3;
```

Comment:
On S to NS exception there may be more registers, but that doesn't seem to be our case.